TestApp/src/main/java/io/ionic/portals/testapp/ViewModelActivity.kt

```diff
@@ -12,7 +12,7 @@ class ViewModelActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        val viewModel: MyPortalViewModel by viewModels()
+        //val viewModel: MyPortalViewModel by viewModels()
```

Comment:
Do we want to leave this commented out or remove it?

Reply:
Removed